.github/workflows/pull_request.yml

```diff
@@ -41,9 +28,27 @@ jobs:
           reporter: github-pr-review
           path: '.'
           pattern: '*.sh'
+          
+      - uses: cachix/install-nix-action@v22
+      - name: Check if the chart follows a valid semver version
+        run: |
+          export NIX_PATH=nixpkgs=$(jq '.nixpkgs.url' scripts/nix/sources.json -r)
+          echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
```

Comment:
I'd suggest setting up the nix in separate run altogether so it's easy to read and copy&paste to other actions as needed

Reply:
As in like the prepoplulate step.

Reply:
Yes